### PR TITLE
Add sudo prerequisite for agents

### PR DESCRIPTION
**Pull request by abramso**
Added a prerequisite for sudo

### CODE REVIEW

File: content/agents/overview.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ However, you should be aware of certain implications:
 
 * SSH port (22 by default) should be open for incoming connections.
 * An SSH server should be running on the host.
+* The agent user must have sudo privileges.
```
Comment (idanmo) [inline content/agents/overview.md]: sudo is required in general, not only for remote agent installation...

Comment (abramso) [inline content/agents/overview.md]: Not sure what's wrong with the wording, The "agent user" refers to the user running on the VM, not only for remote installation but also for running the agent services.

Comment (idanmo) [inline content/agents/overview.md]: The section "Pre-requisites for Init Script and Provided Agent Installations" should also contain the sudo requirement for installations on Linux.

**Comment by abramso** [comment]
dd sudo prerequisite for Init-script

**Comment by nir0s** [comment]
What's the status on this?

**Pull request merged**